docs/implementations.rst

```diff
@@ -0,0 +1,75 @@
+=========================
+Saliency Implementations
+=========================
+
+Included with this toolkit are a number of implementations for the interfaces
+described in the previous section.
+Like those interfaces, these initial implementations focus of black-box visual
+saliency and are separated by use-case.
+Each implementation provides functionality for a different perturbation or
+heatmap generation algorithm specific to that use-case.
```

Comment:
I was trying to get at the fact that they are inherently divided into use-cases because the interfaces that they implement are. But I guess that's pretty obvious and what you wrote portrays the intent of the implementations more clearly. Thanks, I'll change it.